src/main/java/com/dislinkt/post/dto/CommentDTO.java

```diff
@@ -0,0 +1,61 @@
+package com.dislinkt.post.dto;
+
+import java.util.UUID;
+
+import javax.validation.constraints.NotBlank;
+
+public class CommentDTO {
+    
+    private Integer id;
+
+    @NotBlank(message = "You can't leave an empty comment!")
+    private String text;
+
+    private UUID personId;
+
+    private Integer postId;
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public String getText() {
+        return text;
+    }
+
+    public void setText(String text) {
+        this.text = text;
+    }
+
+    public UUID getPersonId() {
+        return personId;
+    }
+
+    public void setPersonId(UUID personId) {
+        this.personId = personId;
+    }
+
+    public Integer getPostId() {
+        return postId;
+    }
+
+    public void setPostId(Integer postId) {
+        this.postId = postId;
+    }
+
+    public CommentDTO() {
+    }
+
+    public CommentDTO(Integer id, @NotBlank(message = "You can't leave an empty comment!") String text, UUID personId,
```

Comment:
Do you need constraint validation here since you already have it above?

Reply:
It got generated here automatically because I had it above (I guess that's how it works in vscode)